packages/fx-core/src/plugins/resource/bot/resources/strings.ts

```diff
@@ -72,6 +72,7 @@ export class PluginBot {
   public static readonly VALID_DOMAIN = "validDomain";
   public static readonly BOT_CHANNEL_REGISTRATION = "botChannelReg";
   public static readonly BOT_WEB_APP_RESOURCE_ID = "botWebAppResourceId";
```

Comment:
Since we have another resourceId, do we still need this one "botWebAppResourceId" ?